bubblewrap.c

```diff
@@ -1764,6 +1890,9 @@ main (int    argc,
 
   parse_args (&argc, &argv);
 
+  if ((requested_caps[0] || requested_caps[1]) && is_privileged && getuid () != 0)
```

Comment:
This check seems wrong. is_privileged is only ever true if we're setuid root, but not running as root. In the case when getuid() return 0, we would not have set is_privileged. So, we could just drop the getuid() check here, no?